docs/src/main/sphinx/udf/function.md

```diff
@@ -11,6 +11,7 @@ FUNCTION name ( [ parameter_name data_type [, ...] ] )
   [ CALLED ON NULL INPUT ]
   [ SECURITY { DEFINER | INVOKER } ]
   [ COMMENT description]
+  [ WITH properties AS block]
   statements
```

Comment:
This is the new syntax: either SQL routine statements or a function definition as a dollar-quoted string:
```
{ statements | AS definition }